EIPS/eip-2045.md

```diff
@@ -0,0 +1,119 @@
+---
+eip: 2045
+title: Fractional gas costs for EVM opcodes
+author: Casey Detrio (@cdetrio)
+discussions-to: https://ethereum-magicians.org/t/eip-2045-fractional-gas-costs/3311
+status: Draft
+type: Standards Track
+category Core
+created: 2019-05-17
+---
+
+## Abstract
+According to recent benchmarks, EVM opcodes for computation (ADD, SUB, MUL, etc.) are generally overpriced relative to opcodes for storage I/O (SLOAD, SSTORE, etc.). Currently the minimum gas cost is 1 (i.e. one unit of gas), and most computational opcodes have a cost near to 1 (e.g. 3, 5, or 8), so the range in possible cost reduction is limited. A new minimum unit of gas, called a "particle", which is a fraction of 1 gas, would expand the range of gas costs and thus enable reductions below the current minimum.
```

Comment:
Perhaps I wouldn't say "fraction" but "subdivision of gas".

Reply:
Are they equivalent? If so then fraction is clearer imo (also if the change is made here then the title should change as well).

Reply:
You're right, subdivision is not really used in mathematics/computer science.